settings.gradle

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+rootProject.name = 'bookkeeper'
+
+include('circe-checksum', 'circe-checksum:src:main:circe',
```

Comment:
the modules `circe-checksum:src:main:circe` and `cpu-affinity:src:main:affinity` look like an odd solution. There must be cleaner ways to define these.

Reply:
one possibility is this:
```
include('circe-checksum-jni')
project('circe-checksum-jni').projectDir='circe-checksum/src/main/circe'
```
However, it seems wrong to make 'circe-checksum/src/main/circe' the project directory.

Reply:
This does not work. -include('circe-checksum', 'circe-checksum:src:main:circe',
+include('circe-checksum',
         'cpu-affinity', 'cpu-affinity:src:main:affinity', 'bookkeeper-common',
         'bookkeeper-http:http-server', 'bookkeeper-server',
         'bookkeeper-common-allocator', 'bookkeeper-proto', 'bookkeeper-stats',
         'bookkeeper-stats-providers:prometheus-metrics-provider',
         'tools:framework')
+include(':circe-checksum-jni')
+project(':circe-checksum-jni').projectDir=file('circe-checksum/src/main/circe')

Reply:
I think we can revisit this later. Because I am sure we will encounter some more cases like that down the line.

Reply:
yes, it can be revisited later.